README.md

```diff
@@ -21,19 +21,16 @@ Remix About page
 
 ## About Remix Project
 
-Remix Project is a platform for development tools that use a plugin architecture. It encompasses sub-projects including Remix Plugin Engine, Remix Libraries, and of course Remix IDE.
+The **Remix Project** is a platform for development tools that use a plugin architecture. It encompasses sub-projects including Remix Plugin Engine, Remix Libraries, and of course Remix IDE.
 
-Remix IDE is an open source web and desktop application. It fosters a fast development cycle and has a rich set of plugins with intuitive GUIs. Remix is used for the entire journey of contract development with Solidity language as well as a playground for learning and teaching Ethereum.
+The **Remix IDE** is an open source web and desktop application. It fosters a fast development cycle and has a rich set of plugins with intuitive GUIs. Remix is used for the entire journey of contract development with Solidity language as well as a playground for learning and teaching Ethereum. You can start developing with Remix on the browser by visiting: https://remix.ethereum.org. If you prefer a desktop version, check out our releases: https://github.com/ethereum/remix-desktop/releases.
```

Comment:
Could you put a paragraph break before this sentence "You can start developing with Remix on the browser by..."

and a new paragraph for the Remix Desktop sentence.

Actually for the Desktop sentence, please update it to:
If you prefer a desktop version of Remix, check out: https://github.com/remix-project-org/remix-desktop.

(I'd like it to link to the Desktop Readme page at the new url)